x/posts/legacy/v0.1.0/types.go

```diff
@@ -15,16 +15,16 @@ type GenesisState struct {
 // PostID represents a unique post id
 type PostID uint64
 
-// Post is a struct of a post
+// TextPost is a struct of a post
```

Comment:
Refactoring error, I think this should be `Post` instead

```suggestion
// Post is a struct of a post
```

Reply:
This is still here :stuck_out_tongue_winking_eye: